pkg/corerp/api/v20231001preview/zz_generated_constants.go

```diff
@@ -257,21 +254,21 @@ func PossiblePortProtocolValues() []PortProtocol {
 type ProvisioningState string
 
 const (
-	// ProvisioningStateAccepted - The resource create request has been accepted
+// ProvisioningStateAccepted - The resource create request has been accepted
 	ProvisioningStateAccepted ProvisioningState = "Accepted"
-	// ProvisioningStateCanceled - The resource provisioning has been canceled
+// ProvisioningStateCanceled - The resource provisioning has been canceled
 	ProvisioningStateCanceled ProvisioningState = "Canceled"
-	// ProvisioningStateCreating - The resource is being created
+// ProvisioningStateCreating - The resource is being created
```

Comment:
Comments formatting seems off..

Reply:
yes, I see it. I cannot update it but I can let the team know/file an issue to help update it.

Reply:
sounds good, thanks!